docs/setup_guides/hpc.rst

```diff
@@ -79,7 +97,8 @@ this section.)
 .. note::
 
    When using recent Jupyter iteration the following section can be replaced by simply invoking the command::
```

Comment:
So we should make commands wit plural?

Reading the whole documentation, I think we don't want to ignore the section completly, as the two last steps should be done either case. Maybe we should add a subsection for setting the password?

Reply:
Agreed, and the two last steps are indeed important in any case.